userspace/libsinsp/filterchecks.cpp

```diff
@@ -5235,7 +5239,12 @@ uint8_t* sinsp_filter_check_tracer::extract_args(sinsp_partial_tracer* pae, OUT
 
 	if(m_storage_size < encoded_args_len)
 	{
-		m_storage = (char*)realloc(m_storage, encoded_args_len);
+		char *reduced_storage = (char*)realloc(m_storage, encoded_args_len);
+		if(reduced_storage == NULL)
+		{
+			return NULL;
```

Comment:
Not freeing `m_storage` here?
But we are freeing it right below (in `extract_tracer`).

Reply:
`m_storage` is freed by the destructor 🤔

Reply:
Thanks for checking, I believe we don't need to free `m_storage` in two instances below